ICSharpCode.NRefactory.CSharp/ICSharpCode.NRefactory.CSharp.csproj

```
@@ -23,7 +23,10 @@
     <NoWin32Manifest>False</NoWin32Manifest>
     <IntermediateOutputPath>obj\$(Configuration)\</IntermediateOutputPath>
     <DocumentationFile>$(IntermediateOutputPath)ICSharpCode.NRefactory.CSharp.xml</DocumentationFile>
-  </PropertyGroup>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <EnableDefaultCompileItems>False</EnableDefaultCompileItems>
+    <EnableDefaultEmbeddedResourceItems>False</EnableDefaultEmbeddedResourceItems>
+    <TargetFramework>net472</TargetFramework>  </PropertyGroup>
```

Missing eol here, but not a big deal
```suggestion
    <TargetFramework>net472</TargetFramework>
  </PropertyGroup>
```

---

I can fix while I deal with conflicts.